_posts/en/newsletters/2024-07-12-newsletter.md

```diff
@@ -0,0 +1,132 @@
+  significantly alter the block production rate, so it makes sense to
+  fix this alongside the min-difficulty bug. Since it is also part of
+  the [consensus cleanup soft fork][topic consensus cleanup],
+  test-running the fix in Testnet4 first provides useful early feedback.
+  This PR fixes the time warp bug by checking that the first block of a
+  new difficulty epoch is not earlier than 2 hours before the last block
+  of the previous epoch."
+  a2link="https://bitcoincore.reviews/29775#l-68"
+
+  q3="What is the message in the Genesis block in Testnet 3?"
+  a3="Testnet 3, as well as all other nets (preceding Testnet 4), have
+  the same well-known genesis block message: \"The Times
+  03/Jan/2009 Chancellor on brink of second bailout for banks\".
+  Testnet4 is the first network to have its own genesis block message,
+  which includes the hash of a recent mainnet block (currently
+  `000000000000000000001ebd58c244970b3aa9d783bb001011fbe8ea8e98e00e`) to
+  provide strong guarantees that no pre-mining on this Testnet 4 chain
+  occurred before this mainnet block was mined."
+  a3link="https://bitcoincore.reviews/29775#l-17"
+%}
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Bitcoin Core 26.2][] is a maintenance version of Bitcoin Core's older
+  release series.  Anyone on 26.1 or earlier who unable or unwilling to
+  upgrade to the latest release (27.1) is encouraged to upgrade to this
+  maintenace release.
+
+- [LND v0.18.2-beta][] is a minor release to fix a bug affecting users
+  of older versions of the btcd backend.
+
+## Notable code and documentation changes
+
+_Notable recent changes in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo],
+[Lightning BLIPs][blips repo], [Bitcoin Inquisition][bitcoin inquisition
+repo], and [BINANAs][binana repo]._
+
+- [Rust Bitcoin #2949][] adds a new method `is_standard_op_return()` to validate
+  OP_RETURN outputs against current standardness rules, allowing
+  programmers to test whether OP_RETURN data that exceeds the 80-byte
+  maximum size enforced by Bitcoin Core.  Programmers who aren't worried
+  about exceeding the current default Bitcoin Core limit can continue to
+  use Rust Bitcoin's existing `is_op_return` function.
+
+- [BDK #1487][] introduces support for custom input and output sorting functions
+  by adding a `Custom` variant to the `TxOrdering` enum, to enhance flexibility
+  in transaction construction. Explicit [BIP69][] support is deprecated because
```

Comment:
In original PR it was deprecated but completely removed in later commits.

```suggestion
  in transaction construction. Explicit [BIP69][] support is removed because
```